kerckhoff/packages/models.py

```diff
@@ -0,0 +1,57 @@
+import uuid
+
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from .constants import *
+
+class PackageSet(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    slug = models.SlugField(max_length=32, unique=True) # ??
+    drive_folder_id = models.CharField(max_length=512, blank=True)
```

Comment:
let's get rid of drive_folder_id and drive_folder_url for now - we can keep those in the metadata